docs/demo.md

```diff
@@ -0,0 +1,77 @@
+# MutAcc Demo
+
+To get an intuition over how MutAcc works, this demo provides a fast go through of the main features.
+
+Files that can be used in this demo are found in the demo/ folder in the root of this repository. These files are simulated fastq, and bam-files that represents a father/mother/child trio. The files cover a 2000 bp region of the CFTR gene, with 200 paired reads for each file 2*151bp. The fastq files were generated using [wgsim](https://github.com/lh3/wgsim), and the bam files were generated aligning these fastq-files with [bwa](https://github.com/lh3/bwa) against the [GRCh37](https://www.ensembl.org/info/website/tutorials/grch37.html) reference. The child has a simulated de-novo heterozygous SNV 7:117235031 G->A. This demo shows how this family, along with the causative clinical variant can be uploaded into the mutacc database, and later exported to create a new synthethic sample.
+
+in this demo, the flag ```-d/--demo``` is used after the main command , i.e. ```mutacc --demo [subcommands] [options]```. With this flag, no configuration file will be necessary. This will create the mutacc root directory in the home directory of the user ```~/mutacc_root_demo/``` which can be removed after the demo is made. For this demo to work, it requires that a mongodb process is running on host 'localhost' and 27017. If mutacc is installed on a conda environment (which is recommended!) source the environment before starting this demo. Don't forget to ```cd```into the root directory of this repo.
+
+## Extract reads from case
+
+Before this family can be uploaded into the database, the reads from overlapping the given causative variant (specified in ```demo/variant1.vcf.gz```) must be extracted. This is done with the following command
+
+ ```terminal
+ mutacc --demo extract --case demo/case.yaml --padding 100
```

Comment:
That would be nice. The downside of this would be that the user won't run the demo as it is supposed to be used, i.e. giving the paths to the necessary files. What about just saying in the docs that user needs to give the full path to wherever they have downloaded the repo?